packages/react/src/spotlight-content/SpotlightContent.css.ts

```diff
@@ -0,0 +1,13 @@
+import { recipe, style } from "../vanilla-extract";
+
+export const content = recipe({
+  base: [
+    {
+      overflow: "hidden",
+    },
+    style({
+      top: "12vh",
```

Comment:
Vai, should we use use `vh` or `dvh` here?

Reply:
addressed - also moved all dialogs to top of screen since dialog content can shift and should not change depending on the content (which it does not in main)

and also move alert dialogs to the top third of the screen so it looks better